[2025-02-18T04:31:06.044+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T04:31:06.058+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-03-26T00:00:00+00:00 [queued]>
[2025-02-18T04:31:06.066+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-03-26T00:00:00+00:00 [queued]>
[2025-02-18T04:31:06.067+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-18T04:31:06.079+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): hello_task> on 2024-03-26 00:00:00+00:00
[2025-02-18T04:31:06.088+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=357) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-18T04:31:06.091+0000] {standard_task_runner.py:72} INFO - Started process 363 to run task
[2025-02-18T04:31:06.090+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'hello_task', 'scheduled__2024-03-26T00:00:00+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/my-first-dag.py', '--cfg-path', '/tmp/tmpz57mbyc3']
[2025-02-18T04:31:06.101+0000] {standard_task_runner.py:105} INFO - Job 41: Subtask hello_task
[2025-02-18T04:31:06.134+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:31:06.183+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.hello_task scheduled__2024-03-26T00:00:00+00:00 [running]> on host 3420485c558b
[2025-02-18T04:31:06.186+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:31:06.347+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-26T00:00:00+00:00'
[2025-02-18T04:31:06.352+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T04:31:06.371+0000] {logging_mixin.py:190} INFO - Hello, Airflow!
[2025-02-18T04:31:06.373+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T04:31:06.396+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T04:31:06.403+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=hello_task, run_id=scheduled__2024-03-26T00:00:00+00:00, execution_date=20240326T000000, start_date=20250218T043106, end_date=20250218T043106
[2025-02-18T04:31:06.498+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T04:31:06.527+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-18T16:34:08.870+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T16:34:08.923+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-03-26T00:00:00+00:00 [queued]>
[2025-02-18T16:34:08.948+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-03-26T00:00:00+00:00 [queued]>
[2025-02-18T16:34:08.949+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-18T16:34:09.020+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): hello_task> on 2024-03-26 00:00:00+00:00
[2025-02-18T16:34:09.030+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1639) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-18T16:34:09.051+0000] {standard_task_runner.py:72} INFO - Started process 1645 to run task
[2025-02-18T16:34:09.054+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'hello_task', 'scheduled__2024-03-26T00:00:00+00:00', '--job-id', '408', '--raw', '--subdir', 'DAGS_FOLDER/my-first-dag.py', '--cfg-path', '/tmp/tmp9qtw4n8t']
[2025-02-18T16:34:09.065+0000] {standard_task_runner.py:105} INFO - Job 408: Subtask hello_task
[2025-02-18T16:34:09.117+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T16:34:09.212+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.hello_task scheduled__2024-03-26T00:00:00+00:00 [running]> on host 8720d9c228eb
[2025-02-18T16:34:09.216+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T16:34:09.511+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-26T00:00:00+00:00'
[2025-02-18T16:34:09.514+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T16:34:09.531+0000] {logging_mixin.py:190} INFO - Hello, Airflow!
[2025-02-18T16:34:09.532+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T16:34:09.561+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T16:34:09.566+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=hello_task, run_id=scheduled__2024-03-26T00:00:00+00:00, execution_date=20240326T000000, start_date=20250218T163408, end_date=20250218T163409
[2025-02-18T16:34:09.667+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T16:34:09.774+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-18T16:34:09.778+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
